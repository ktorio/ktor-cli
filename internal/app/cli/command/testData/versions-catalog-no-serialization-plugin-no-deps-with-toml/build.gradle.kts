plugins {
    alias(libs.plugins.kotlin.jvm)
}

repositories {
    mavenCentral()
}
