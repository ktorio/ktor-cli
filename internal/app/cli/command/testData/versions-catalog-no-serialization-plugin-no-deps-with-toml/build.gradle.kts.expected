plugins {
    alias(libs.plugins.kotlin.jvm)
    alias(libs.plugins.kotlin.serialization)
}

repositories {
    mavenCentral()
}

dependencies {
    implementation(libs.ktor.serialization.kotlinx.json)
}
